### DISCUSSION TITLE
Detect Idle mode on Android devices

### DISCUSSION THREAD

**Discussion by idzdigital** [airsdk/Adobe-Runtime-Support#1288] [Q&A]
I would like to if there is any way to detect idle mode on Android devices. Also can I change the idle timeout?

**Comment by itlancer** [comment]
Here feature request for that: https://github.com/airsdk/Adobe-Runtime-Support/issues/108
Right now it possible only by writing native extension.

**Reply by itlancer** [reply]
https://github.com/airsdk/Adobe-Runtime-Support/issues/108
Now it works for Android with latest AIR versions.
